presto-native-execution/presto_cpp/presto_protocol/README.md

```diff
@@ -2,10 +2,11 @@
 
 ### Required Dependencies:
 
-Install [PyYAML](https://pyyaml.org/) and [Chevron](https://github.com/noahmorrison/chevron)
+Install [PyYAML](https://pyyaml.org/), [Chevron](https://github.com/noahmorrison/chevron) and [jq](https://github.com/jqlang/jq)
```

Comment:
Would you extract these changes into a separate PR? CC: @majetideepak

Reply:
Let's move this to a separate PR. `brew jq` is MacOS specific. We should probably update the setup scripts to include jq.

Reply:
https://github.com/prestodb/presto/pull/20052